package.json

```diff
@@ -58,6 +62,7 @@
   ],
   "activationEvents": [
     "onCommand:pacCLI.openDocumentation",
+    "onCommand:microsoft-powerapps-portals.webExtension.init",
```

Comment:
suggestion: until the webExt/VSCodeDev functionality reaches MVP/pubPreview quality and feature set, can we keep this under "feature flags", i.e. make it an opt-in feature for the std VSCode Desktop, WSL and GH Codespaces experiences?
In the past we used the regular extensions settings to make this an opt-in experience for the time being

CC: @tehcrashxor

Reply:
Added fcb.